src/main/kotlin/com/example/plugin_for_ux/utils/Formatter.kt

```diff
@@ -0,0 +1,25 @@
+package com.example.plugin_for_ux.utils
+
+import com.example.plugin_for_ux.checkers.CheckerResult
+
+class Formatter {
+
+    fun formatAnswer(answersList:List<CheckerResult>): List<Pair<String, String>> {
+
+        val pairList = mutableListOf<Pair<String, String>>()
```

Comment:
я бы просто переименовал в `result`
```suggestion
        val result= mutableListOf<Pair<String, String>>()
```